platform/configure/config.mdx

```diff
@@ -79,79 +83,83 @@ helm upgrade loft vcluster-platform --install \
       X-XSS-Protection: 1; mode=block
 ```
 
-### Setting additional certificate authority for Platform
-If you want to have **all** your virtual clusters managed by the vCluster Platform to use a specific certificate authority to establish the connection between the virtual cluster and the platform, you can set it as a configuration option on the vCluster Platform instead of configuring it on each individual virtual cluster.
-This will enable secure communication between your Virtual Clusters and the vCluster Platform.
-As a pre-requisite, you have to [configure TLS for the vCluster Platform](/platform/configure/domain#configure-tls) first.
-```yaml
+### Setting additional certificate authority for platform
+
+If you want to have **all** your virtual clusters managed by the platform use a
+specific certificate authority to establish the connection between the virtual
+cluster and the platform, you can set it as a configuration option on the
+platform instead of configuring it on each individual virtual cluster. This
+enables secure communication between your virtual clusters and the platform.
+
+As a prerequisite, you have to [configure TLS for the platform](/platform/configure/domain#configure-tls) first.
+
+```yaml title="Additional certificate authority configuration"
 additionalCA: "(base64 encoded CA)"
 ```
 
-Also, if you need to import externally managed vCluster to the vCluster Platform, you can also enable a secure connection by grabbing the value from the existing vCluster Platform, you can do this by running:
+If you need to import externally managed virtual clusters to the platform, you can also enable a secure connection by grabbing the value from the existing platform. You can do this by running:
 
-```shell
+```bash title="Get additionalCA value"
 helm get values --namespace [namespace where vCluster Platform is deployed] [loft helm release name] --all | grep additionalCA
 ```
 
-Using the `additionalCA` value from the above command, import the vCluster using the following vCluster CLI command:
-```shell
-  vcluster platform add vcluster <VCLUSTER_NAME> \
+Using the `additionalCA` value from the preceding command, import the vCluster using the following vCluster CLI command:
+
+```bash title="Import vCluster with additionalCA"
+vcluster platform add vcluster <VCLUSTER_NAME> \
 --namespace=<VCLUSTER_NAMESPACE> \
 --project=<PROJECT_NAME> \
 --ca-data [base64 string grabbed from the command above]
 ```
 
 Once the command succeeds, the imported vCluster pod restarts and reports `Ready`.
 
+### The platform agent
 
-### The vCluster Platform Agent
-
-Every connected cluster in a vCluster Platform environment, including the "main" cluster that vCluster Platform is
-deployed in, must have a vCluster Platform Agent deployed. This agent pod handles reconciliation of cluster
-scoped (as in not vCluster Platform instance wide) resources.
+Every connected cluster in a platform environment, including the "main" cluster
```

Comment:
why is Agent capitalized through here? Also, I think on the cluster, it's called `loft` without any reference that it is an agent.